commitizen/commands/undo.py

```diff
@@ -0,0 +1,61 @@
+from commitizen import cmd, factory, git, out
+from commitizen.config import BaseConfig
+from commitizen.exceptions import InvalidCommandArgumentError
+
+
+class Undo:
+    """Reset the latest git commit or git tag."""
```

Comment:
It seems we use various terms for the same meaning (i.e., reset / revert / undo). I'll suggest unifying them.

Reply:
I could use "undo". 👌